### DISCUSSION TITLE
Error running rna-Spades

### DISCUSSION THREAD

**Discussion by d-kk** [ablab/spades#928] [Q&A]
### Description of bug

While running rnaSpades we see this warning, previously have seen this error and the program executed to completion. 

Is the rnaSpades output valid in this case? Should we be concerned about the output?

Thanks and lmk if you need more info.

0:20:02.343    10M / 65G   INFO    General                 (kmer_splitter.hpp         : 101)   Using cell size of 17476
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304

### spades.log

Run did not fail, is program output valid given this error:

0:20:02.343    10M / 65G   INFO    General                 (kmer_splitter.hpp         : 101)   Using cell size of 17476
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304
mimalloc: warning: unable to allocate from region: size 4194304

### params.txt

Run did not fail, pl see above.

### SPAdes version

3.15.4

### Operating System

ubuntu

### Python Version

2.7.17

### Method of SPAdes installation

binaries

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Hello

These warnings are benign. You can ignore them
